include/openmc/distribution.h

```diff
@@ -98,6 +98,45 @@ class Watt : public Distribution {
   double b_; //!< Factor in square root [1/eV]
 };
 
+//==============================================================================
+//! Gaussian (fusion) spectrum with form 1/std_dev*sqrt(pi) exp (-(e-E0)/std_dev)^2
```

Comment:
I would prefer calling this `Normal` instead of `Gaussian`. Also, in the equation for the PDF, I'm used to seeing `sqrt(2*sigma^2)` in the denominator outside the `exp`, and `2*sigma^2` in the denominator inside the `exp`. Can you explain why there's a difference?

Reply:
I think this might be a combination of a misnaming and an error, I was sampling for Muir using equation 8 from https://permalink.lanl.gov/object/tr?what=info:lanl-repo/lareport/LA-05411-MS - and incorrectly conflating std_dev with W. Ill fix this and then push. I reckon when I push the unit tests that would've been caught by the tests I was planning :)

Reply:
Yeah, just misnaming and an error in the the way Muir was called.